Patches/RimTraits - Medieval Talents/RT_Traits.xml

```diff
@@ -0,0 +1,255 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>RimTraits - Medieval Talents</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+
+				<!-- Tier 1 -->
+				<li Class="PatchOperationRemove">
```

Comment:
Any reason these pairs of Remove and subsequent Add operations can't be replaced with a single Replace operation?

Reply:
I wasn't sure about using replace instead of remove and add the moment I write the patch, but if Replace operations are intended to reduce clutter then I'll try to change how the patch applied